src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { Button } from '../Button/Button';
+
+import { Modal } from './Modal';
+
+storiesOf('Modal', module)
+	.addParameters({ jest: ['Modal', 'ModalBackdrop'] })
+	.add('Modal', () => (
+		<Modal
+			isOpen={true}
+			title="Modal title"
+			renderFooterLeft={() => <Button type="secondary" label="Cancel" />}
```

Comment:
zouden we dit niet implementeren als meerdere slots?
eg: https://medium.com/@srph/react-imitating-vue-slots-eab8393f96fd